assets/custom.css

```diff
@@ -118,5 +138,9 @@ html {
         background-color: white;
     }
 }
-  
+@media (max-width: 1024px) {
+    .columns {
+        display: block !important;
```

Comment:
The use of `!important` should be always avoid it. If you can get it done without it don't use it.

In this case it's really simple to do so, just configure the grid to only show a column.